package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "to-do-list",
+  "version": "1.0.0",
+  "description": "",
+  "main": "webpack.config.js",
+  "scripts": {
+    "build": "webpack --config webpack.config.js"
+  },
```

Comment:
- [ ] as the previous CR already mentioned according to the requirements we need to include scripts for building Webpack both in development and production mode. this might look something like below 
```
"start": "webpack serve --open"
```